src/kbmod/search/trajectory_list.h

```diff
@@ -29,6 +29,7 @@ class TrajectoryList {
     TrajectoryList(const TrajectoryList&) = delete;
     TrajectoryList& operator=(TrajectoryList&) = delete;
     TrajectoryList& operator=(const TrajectoryList&) = delete;
+    TrajectoryList& operator=(TrajectoryList&&) noexcept;
```

Comment:
As I noted in another comment, I think it would be cleaner to just use `set_trajectories` instead of the move operator.

Reply:
Removed in favour of `set_trajectories`